src/main/java/com/indore/resources/ImageResource.java

```diff
@@ -1,9 +1,110 @@
 package com.indore.resources;
 
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import org.glassfish.jersey.media.multipart.FormDataParam;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.*;
+import java.net.URL;
+
 /**
  * Resource for user's album.
  *
  * @author Amit Khandelwal
  */
+@Path("/image")
 public class ImageResource {
-}
+    private static final Logger log = LoggerFactory.getLogger(UserResource.class);
+
+    String accessKey;
+    String secretAccesKey;
+    public ImageResource(String accessKey, String secretAccesKey) {
+        this.accessKey=accessKey;
+        this.secretAccesKey=secretAccesKey;
+    }
+    @POST
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @Path("/upload")
+    public Response imageUpload(@FormDataParam("file") final InputStream inputStream) {
+       String uploadedFileLocation="";
+        try{
+            //Double fileName= Math.floor(Math.random() * 90000000);
```

Comment:
Always add `TODO` for work, which you intend to do in future, You can easily check all `TODO` in your IDE.